dashboard/config/initializers/devise.rb

```diff
@@ -130,7 +130,7 @@
 
   # ==> Configuration for :rememberable
   # The time the user will be remembered without asking for credentials again.
-  # config.remember_for = 2.weeks
+  config.remember_for = 2.weeks
```

Comment:
hah, did this ever work?  (from history, it looks like maybe no, or at least not for a long time)